layers/+tools/lsp/funcs.el

```diff
@@ -46,3 +46,98 @@ https://github.com/emacs-lsp/lsp-javascript/issues/9#issuecomment-379515379"
       (all-completions (company-grab-symbol) candidates)))
   (make-local-variable 'company-transformers)
   (add-to-list 'company-transformers 'lsp-prefix-company-transformer))
+
+(defun spacemacs/lsp-bind-keys-for-mode (mode)
+  "Define key bindings for the specific MODE."
+  (spacemacs/declare-prefix-for-mode mode "m=" "format")
+  (spacemacs/declare-prefix-for-mode mode "mg" "goto")
+  (spacemacs/declare-prefix-for-mode mode "mh" "hierarchy")
```

Comment:
IMO this shouldn't be here since only small part of the LSP server provide hierarchy support. Also, as per  CONVENTIONS.org "mh" stands for help. I am voting to remove this line out of base layer and maybe use "mH" for hierarchy in C++ mode. @sdwolfz what do you think?

Reply:
I've no objections, but maybe move this discussion over to #11203 (which I'll re-title) as this PR is now closed